packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -38,6 +38,8 @@ class IconButton extends StatelessWidget {
   const IconButton({
     Key key,
     this.size: 24.0,
+    this.padding: const EdgeInsets.all(8.0),
+    this.alignment: FractionalOffset.centerLeft,
```

Comment:
i'm surprised that it even matters in the appbar. don't we force the width enough?